compiler/test/stdlib/array.test.gr

```diff
@@ -224,17 +224,18 @@ let arrB = [> 2, 3, 4]
 let arrC = [> 1]
 let arrD = [> 2]
 
-assert Array.product(arrA, arrB) == [>(1,2),(1,3),(1,4),(2,2),(2,3),(2,4)]
-assert Array.product(arrC, arrB) == [>(1,2),(1,3),(1,4)]
-assert Array.product(arrC, arrD) == [>(1,2)]
+assert Array.product(arrA, arrB) ==
+[> (1, 2), (1, 3), (1, 4), (2, 2), (2, 3), (2, 4)]
```

Comment:
It's annoying that this doesn't indent. I believe @ospencer pointed that out in #1390